challenges/objects-arrays.js

```diff
@@ -48,22 +92,36 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
+
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+
+const universities = [
+  graduates.map(function(item){
+  return item["university"]}).sort()
+  ];
+
+console.log(universities);
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = [
+  graduates.map(function(item){
+    return `${item['first_name']} ${item['email']}`;
+  })
+];
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = [
+  graduates.filter(function(item) {
+    if (item['university'].includes('Uni')) {return true;}
```

Comment:
Good job sort of hacking this one together, but as we know now that we've gone through it, filter has an if statement built in. And if the condition in the filter's callback function body is true, then the item gets passed into the new array, if it's not true, then it gets filtered.

Reply:
Also good job using your problem solving skills to find the .includes method.